File: 217-contains-duplicate/contains-duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        set<int>s;
        for(auto &n: nums){
            s.insert(n);
        }
        return !(s.size()==nums.size());
    }
};
// Time Complexity O(n)
// Space Complexity O(n)

File: 217-contains-duplicate/contains-duplicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
    let arr=[];
    for(let num of nums){
        if(arr[num])return true;
        else arr[num]=true;
    }
    return false;
};